recipe/meta.yaml

```diff
@@ -70,6 +70,10 @@ outputs:
         - cmake
         - make     # [unix]
         - ninja    # [win]
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
whoa, magic, is this documented?

Reply:
There's a blog post, https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/